src/cf-definitions-builder.ts

```diff
@@ -50,6 +50,12 @@ export default class CFDefinitionsBuilder {
         return this;
     };
 
+    public appendTypes = (models: CFContentType[]): CFDefinitionsBuilder => {
```

Comment:
can we name the param `contentTypes`?

Reply:
It's not a coincidence, `appendType` argument is called `model`. Should I change them both?